libr/io/p/io_gdb.c

```diff
@@ -8,6 +8,7 @@
 #define IRAPI static inline
 #include <libgdbr.h>
 #include <gdbclient/commands.h>
+#include <r_core.h>
```

Comment:
you cant do this, rcore depends on rio. rio CANT depend on rcore.

use rcorebind like i have explained, if not i can do this change and y ou can base your pr on top of this

Reply:
I'm not sure to understand what you are asking for with rcorebind. Should I add a new function pointer for sections access ?